theme/base-2016/extrafields.twig

```diff
@@ -4,10 +4,10 @@
 
     <h1>{{ record.title }}</h1>
 
-    {# Output all fields, in the order as defined in the contenttype.
-       To change the generated html and configure the options, see:
-       https://docs.bolt.cm/templating #}
+    {# Using the 'with' tag with the 'only' keyword, we limit the variable scope inside the block #}
+    {% with { 'record': record, 'common': false, 'repeaters': true } only %}
```

Comment:
So, either we pass in `global`, or we drop the `only`, because we need to get the `_route` in the block.